samples/java-eventing-shopping-cart/src/main/proto/cart/shopping_cart_topic.proto

```diff
@@ -33,6 +33,9 @@ message TopicOperation {
 }
 
 service ShoppingCartTopicService {
+    option (akkaserverless.service) = {
+        type : SERVICE_TYPE_ACTION
```

Comment:
Do we have a protobuf style guide somewhere? I think I'd use 2-space indents and no space between `type :`, but I'm not sure if we decided on anything. Anyway perhaps not for this PR.

Reply:
I guess we should (at some point) lint these like we do in the framework project.